swagger2.0/oic.r.rule.swagger.json

```diff
@@ -0,0 +1,371 @@
+          {"$ref": "#/parameters/interface-all"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "Retrieves the rule as Links List.",
+            "x-example":
+            [
+              {
+                "href": "/ruleinputcollection",
+                "rt":   ["oic.r.rule.inputcollection"],
+                "if":   ["oic.if.ll","oic.if.baseline"],
+                "p":    {"bm": 3},
+                "eps": [
+                    {"ep": "coaps://[fe80::b1d6]:1111"}
+                ]
+              },
+              {
+                "href": "/ruleexpression",
+                "rt":   ["oic.r.rule.expression"],
+                "if":   ["oic.if.rw","oic.if.baseline"],
+                "p":    {"bm": 3},
+                "eps": [
+                    {"ep": "coaps://[fe80::b1d6]:1111"}
+                ]
+              },
+              {
+                "href": "/ruleactioncollection",
+                "rt":   ["oic.wk.rule.actioncollection"],
+                "if":   ["oic.if.ll","oic.if.baseline"],
+                "p":    {"bm": 3},
+                "eps": [
+                    {"ep": "coap://[fe80::b1d6]:1111"}
+                ]
+              }
+            ],
+            "schema": { "$ref": "#/definitions/slinklist" }
+          }
+        }
+      }
+    },
+    "/RuleResURI?if=oic.if.baseline": {
+      "get": {
+        "description": "A Rule is a Collection made up of 3 Links: \n- A Link to an instance of a Collection of Rule Inputs. \n- A Link to a Rule Expression (the logic of the Rule). \n- A Link to a Collection of Rule Actions.",
+        "parameters": [
+          {"$ref": "#/parameters/interface-all"}
+        ],
+        "responses": {
+          "200": {
+            "description" : "Retrieves the baseline response for the rule.",
+            "x-example":
+            {
+              "rt": ["oic.r.rule"],
+              "if": ["oic.if.ll","oic.if.baseline","oic.if.create"],
+              "rts": ["oic.r.rule.inputcollection","oic.r.rule.expression","oic.r.rule.actioncollection"],
+              "links": [
+                {
+                  "href": "/ruleinputcollection",
+                  "rt":   ["oic.r.rule.inputcollection"],
+                  "if":   ["oic.if.ll","oic.if.baseline"],
+                  "p":    {"bm": 3},
+                  "eps": [
+                    {"ep": "coaps://[fe80::b1d6]:1111"}
+                  ]
+                },
+                {
+                  "href": "/ruleexpression",
+                  "rt":   ["oic.r.rule.expression"],
+                  "if":   ["oic.if.rw","oic.if.baseline"],
+                  "p":    {"bm": 3},
+                  "eps": [
+                    {"ep": "coaps://[fe80::b1d6]:1111"}
+                  ]
+                },
+                {
+                  "href": "/ruleactioncollection",
+                  "rt":   ["oic.r.rule.actioncollection"],
+                  "if":   ["oic.if.ll","oic.if.baseline"],
```

Comment:
Add "oic.if.create"?

Reply:
No.

Reply:
So as not to lose this conversation and possible change, this depends on whether or not the CR is changed to remove oic.if.create.